wagtail/contrib/redirects/filters.py

```diff
@@ -26,3 +27,7 @@ def filter_type(self, queryset, name, value):
         if value and self.request and self.request.user:
             queryset = queryset.filter(is_permanent=value)
         return queryset
+
+    class Meta:
+        model = Redirect
+        exclude = []
```

Comment:
Without this, the tests fail when calling `self.filters.get_fields()` in the generic `IndexView`: https://github.com/wagtail/wagtail/runs/7384787027?check_suite_focus=true